internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -10,25 +14,90 @@ PublicNetworkLoadBalancer:
         - Fn::ImportValue:
             !Sub '${AppName}-${EnvName}-PublicSubnets'
     Type: network
+{{- if .NLB.UDPListenerExists }}
+    SecurityGroups:
+      - !GetAtt NetworkLoadBalancerSecurityGroup.GroupId
+NetworkLoadBalancerSecurityGroup:
+  Metadata:
+    'aws:copilot:description': 'A security group for your network load balancer allowing public traffic'
+  Type: AWS::EC2::SecurityGroup
+  Properties:
+    GroupDescription: Allow traffic to the public facing load balancer on listener ports
+    SecurityGroupIngress:
+{{- range $listener := .NLB.Listener }}
+      - CidrIp: 0.0.0.0/0
```

Comment:
should this not be https://aws.github.io/copilot-cli/docs/manifest/environment/#http-public-ingress-source-ips?

Reply:
I don't think we should specify this here when the `http` field is for the application load balancer. Maybe this should be a feature request for the nlb.

Reply:
Made issue: #5307 👍